dev.c

```diff
@@ -1079,6 +1079,15 @@ __acquires(fc->lock)
 	end_requests(fc, &fc->processing);
 }
 
+void fuse_abort_all_ios(struct fuse_conn *fc)
+{
+	printk(KERN_INFO "PXD_IOCTL : Aborting all requests...");
```

Comment:
Given the context and usage of this function, it should be in pxd.c alongside pxd_abort_context()?

Reply:
end_queued_requests function is static and defined in dev.c, so I had to create another function in the same file to export it.

in v2.5.0, we have pxd_timeout(), which calls fuse_abort_conn() in dev.c, that in turn calls end_queued_requests()

I will leave this as is.